scripts/extract_mecas.sh

```diff
@@ -67,20 +67,19 @@ for file in $INCOMING_DIR/*; do
 
     echo "$uuid" >"$outputDir/source.txt"
 
-    if [ "$doiPrefix" = "10.1101" ]; then
-        pull_docker_image
-        echo "correct some bioRxiv/Encoda XML issues and store $outputDir/$id.xml..."
-        cat "$tmpDir/$xmlFile" | docker run --rm -i ghcr.io/elifesciences/enhanced-preprints-biorxiv-xslt:latest /app/scripts/transform.sh --doi $doiSuffix >"$outputDir/$id.xml"
-        echo "transform.sh --doi $doiSuffix successfully run"
-    else
-        echo "Skipping XML correction for DOI prefix $doiPrefix"
-        cp "$tmpDir/$xmlFile" "$outputDir/$id.xml"
-    fi
-
-    echo "copy all tif, gif and jpg content to ${outputDir}..."
-    cp $tmpDir/content/*.tif "$outputDir/" || true
-    cp $tmpDir/content/*.gif "$outputDir/" || true
-    cp $tmpDir/content/*.jpg "$outputDir/" || true
+    pull_docker_image
+    echo "correct some bioRxiv/Encoda XML issues and store $outputDir/$id.xml..."
+    cat "$tmpDir/$xmlFile" | docker run --rm -i ghcr.io/elifesciences/enhanced-preprints-biorxiv-xslt:latest /app/scripts/transform.sh --doi $doiSuffix >"$outputDir/$id.xml"
+    echo "transform.sh --doi $doiSuffix successfully run"
+    
+    echo "getting image paths from $tmpDir/manifest.xml ..."
+    images=$(cat $tmpDir/manifest.xml | sed 's/xmlns=".*"//g' | xmllint -xpath '//manifest/item[@type="figure"]/instance[starts-with(@media-type,"image")]/@href' -)
```

Comment:
I need to understand why this step is necessary. Can we not just copy all images across from the extracted meca files rather than navigating the XML?

Reply:
The current script assumes all the image files are in a folder titled 'content'. This is convention that bioRxiv use, but it's not in the meca spec. The files could not be placed in such a folder, or the folder could be called something else (as in the files provided by Exeter). 

We need to account for this, and not rely on a convention used by bioRxiv (which they could change).

Reply:
If we intend to use the meca format, then we should consume it according to [the spec](https://groups.niso.org/higherlogic/ws/public/download/23902/NISO_RP-30-2020_Manuscript_Exchange_Common_Approach_MECA.pdf), which states that all files should be listed in the `manifest.xml` file. We should be using that file to retrieve all the article/image/ancillary files we need. The above change is an attempt to do that for image files. 

When we add support for other kinds of files (supplementary files and so on), we'll need a similar approach.